lib/y2_logs_helper.pm

```diff
@@ -0,0 +1,128 @@
+package y2_logs_helper;
+use base "installbasetest";
```

Comment:
If it is a helper package, I guess it should not be inherited from installbasetest. The logs may be related to both installation and YaST module testing.

Reply:
yes but there are still places that y2_logs_helper is used as base class. because those cases are not our responsibility, they have just replaced with the renamed class.

Reply:
fixed